src/components/actionSheet/index.js

```diff
@@ -159,7 +159,7 @@ export default class ActionSheet extends BaseComponent {
 
   renderSheet() {    
     return (
-      <View bg-white>
+      <View testID={this.props.testID} bg-white>
```

Comment:
the task was to add testID to the action sheet options (:

Reply:
@M-i-k-e-l 
I remember talking about this PR, is it possible you didn't push the changes?

Reply:
I did push it, not sure why this has not changed but the [code](https://github.com/wix/react-native-ui-lib/pull/418/files) has changed

Reply:
Look at - "File Changed" tab. 
There's only one file that has changed, weird..

Reply:
I did not push the testID input in the screen, should I have?